src/DynamoCore/DynamoCore.csproj

```diff
@@ -158,10 +149,6 @@
     <Copy SourceFiles="$(ProjectDir)\BuiltInAndOperators\Operators.xml" DestinationFolder="$(OutputPath)\$(UICulture)" />
     <Copy SourceFiles="$(ProjectDir)\BuiltInAndOperators\BuiltIn.xml" DestinationFolder="$(OutputPath)\$(UICulture)" />
     <Copy SourceFiles="$(ProjectDir)\BuiltInAndOperators\BuiltIn.Migrations.xml" DestinationFolder="$(OutputPath)" />
-    <!--Enable these if/when we need to keep old binaries for patch installer compatability-->
-    <Copy SourceFiles="@(LegacyBinariesToBinFolder)" DestinationFolder="$(OutputPath)" Condition=" '$(TargetFramework)' == 'net48' " />
-    <Copy SourceFiles="@(LegacyBinariesToNodesFolder)" DestinationFolder="$(OutputPath)\nodes\" Condition=" '$(TargetFramework)' == 'net48' " />
-     <Copy SourceFiles="@(LegacyBinariesToBinResourcesFolder)" DestinationFolder="$(OutputPath)\en-US\" Condition=" '$(TargetFramework)' == 'net48' " />
```

Comment:
hmm Why are these part of this PR? If we want to remove this step, do we want to remove the corresponding binaries from the legacy folder or even the folder itself?

Reply:
I can remove the legacy folder as well.

Reply:
This strictly doesn't need to belong to this PR but I thought why not clean up this file as I was updating it anyway.